src/WebJobs.Script.WebHost/Resources/Functions/WarmUp/run.csx

```diff
@@ -1,9 +1,26 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿#r "Newtonsoft.Json"
+#r "Microsoft.WindowsAzure.Storage"
+
+
+using System.Net;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Primitives;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Newtonsoft.Json;
 
 public static IActionResult Run(HttpRequest req, TraceWriter log)
 {
     log.Info("WarmUp function invoked.");
 
-    return (ActionResult)new OkObjectResult("WarmUp complete.");
+    string name = req.Query["name"];
 
-}
\ No newline at end of file
+    string requestBody = new StreamReader(req.Body).ReadToEnd();
```

Comment:
The incoming warmup requests don't actually ever include a name right? I'd just remove this and use a constant value below.

Reply:
Yes but I wanted to warmup stream and bunch of other objects as well. The result is ignored.

Reply:
ok